packages/atlas/src/providers/uploads/uploads.hooks.ts

```diff
@@ -176,17 +176,12 @@ export const useStartFileUpload = () => {
         setAssetStatus({ lastStatus: 'error', progress: 0 })
 
         const axiosError = e as AxiosError
-        const networkFailure = axiosError.isAxiosError && !axiosError.response?.status
-        const providerFailure =
+        const networkFailure =
           axiosError.isAxiosError &&
-          axiosError.response &&
-          axiosError.response.status >= 400 &&
-          axiosError.response.status <= 500
-        if (providerFailure) {
-          SentryLogger.error('Failed to upload asset: provider failure', 'uploadsHooks', e, {
-            asset: { dataObjectId: asset.id, uploadOperator },
-          })
-          UserEventsLogger.logDistributorError({ dataObjectId: asset.id, distributorId: uploadOperator.id }, e)
+          (!axiosError.response?.status || (axiosError.response.status >= 400 && axiosError.response.status <= 500))
```

Comment:
`(!axiosError.response?.status || (axiosError.response.status >= 400 && axiosError.response.status <= 500))`

Should we consider codes >500 as a network failure as well and change the provider?

Reply:
@WRadoslaw As I look at this I don't think we're really able to differentiate between errors caused by a user and those caused by the operator. Do you think we need this `if (networkFailure) {` at all?

Reply:
@kdembler I'd say that 500+ responses should definitely trigger blacklisting.

I asked GPT for a snippet as possible handling
```
const makeRequest = async () => {
    try {
        const response = await axios.get('https://example.com/data');
        console.log(response.data);
    } catch (error) {
        if (axios.isAxiosError(error)) {
            const axiosError = error as AxiosError;
            if (axiosError.response) {
                if (axiosError.response.status >= 500) {
                    // Here we can ban storage operator and retry upload
                    console.error('Server error (500+):', axiosError.response.status);
                } else {
                    // Under 500, we cannot distinguish between client and server error, but we need to agree
                    // whether we should try next operator or retry to the same
                    console.error('Unexpected status code:', axiosError.response.status);
                }
            } else if (axiosError.request) { 
                // The request was made but no response was received
                // Generated by GPT - not sure what this would imply - I'd just remove this
                // Theoretically, if there is no response it means that req was rejected or there is a network error
            } else if (axiosError.code === 'ERR_NETWORK') {
                // I'd say here, we should ban operator as well
                console.error('Network error, possibly a CORS issue:', axiosError.message);
            } else {
                // Generated by GPT - not sure what this would imply - I'd just remove this
                console.error('Error setting up the request:', axiosError.message);
            }
        } else {
            // Non-Axios error
            console.error('Non-Axios error:', error);
        }
    }
};
```

Reply:
Yeah, my point is that we may want to consider _any_ error as a basis for blacklisting. Is there something against that?